### ISSUE TITLE
Win 10 - ver 10.0.17763.771 - Fully supported - RDPcheck works but mstsc doesn't work

### ISSUE THREAD

**Issue by livol22** [stascorp/rdpwrap#1001]
As in the title, everything looked fine until i try to connect with mstsc.exe ( tried with localhost / 127.0.0.1 / 127.0.0.2 ) . I can connect with rdpchecker no problem however.

**Comment by livol22** [comment]
here's my rdpconf 
![image](https://user-images.githubusercontent.com/16581569/74740723-be835100-525b-11ea-8303-778f990af2cf.png)

**Comment by livol22** [comment]
I can have multiple RDP sessions but I can't open rdp sessions from within the main rdp session ( like RDPCheck.exe does )

**Comment by mathewjpotter** [comment]
im having this problem as well

**Comment by mathewjpotter** [comment]
I installed hamachi on the server routes the connection that way, it's a temp fix... waiting for input:)

**Comment by chocolait** [comment]
look at #945

**Comment by livol22** [comment]
I updated my rdpwrap.ini file like mentionned but I get the same error message

[rdpwrap.zip](https://github.com/stascorp/rdpwrap/files/4231714/rdpwrap.zip)

**Comment by mathewjpotter** [comment]
Ya, its strange it only affects external IP addresses, not the local ones, thats why hamachi works. Its been working good with hamachi, but I've spent 8hrs plus using different config files and termsrv.dll trying to correct the problem. I even ran some windows updates and sfc /scannow

**Comment by livol22** [comment]
Installing hamachi makes you able to run mstsc 127.0.0.2 ?

**Comment by mathewjpotter** [comment]
thats right, its like a vpn network adapter. its been around for a long time, we've been up and running with that.

**Comment by livol22** [comment]
How do you make it work with hamachi ? Trying to figure out but i'm quite the newbie

**Comment by mathewjpotter** [comment]
you need to download it onto both machines, when you install it on the windows 10 machine, create a network, then on your client machine install hamachi there and join the network you created https://hamachi.en.softonic.com/ Once installed, then copy the hamachi Ip address and use that to connect via RDP, besure to use that link, cause theres new hamachi from logmein thats a different tool, (i think they have a couple softwares now)
